CONTRIBUTING.md

```diff
@@ -91,6 +91,19 @@ with open(pathlib.Path("dump_schedules.json"), "r") as backup:
 
 You're all set!
 
+## backend database schema
+
+The schema of the backend database is stored in dispatcher/backend/docs/schemaspy. It is based
+on a static website generated by SchemaSpy.
+
+To reflect DB schema changes to the doc, you can run the following command once your 
+docker-compose setup is ready (and supposing that your local DB is up-to-date):
+
+```
+cd dispatcher/backend
+docker run --network zimfarm_default -v "$(pwd)/docs/schemaspy.properties:/schemaspy.properties" -v "$(pwd)/docs/schemaspy:/output" schemaspy/schemaspy:latest
+```
+
```

Comment:
I moved this to `dev/README.md` instead since this assume that you have the docker-compose setup ready.
I added instructions regarding alembic and tests